src/api/app/controllers/staging/workflows_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Staging::WorkflowsController < ApplicationController
+  before_action :require_login
+  before_action :set_project
+
+  def create
+    staging_workflow = @project.build_staging
+    authorize staging_workflow
+
+    staging_workflow.managers_group = Group.find_by(title: xml_hash)
+
+    unless staging_workflow.managers_group
+      render_error(
+        status: 404,
+        errorcode: 'not_found',
+        message: "Managers Group '#{params[:managers_title]}' not found."
+      )
+      return
+    end
+
+    if staging_workflow.save
+      render_ok
+    else
+      render_error(
+        status: 400,
+        errorcode: 'invalid_request',
+        message: "Staging for #{@project} couldn't be created"
```

Comment:
Because?

Reply:
It is the same message as in the WebUI. I agree that the reason should be there, but we also want to keep then consistent

Reply:
No we don't. The API is for programs.

Reply:
Same here...

Reply:
I mean that we have the same error message there. You were complaining about the error message, right?